exporter/signalfxexporter/translation/translator.go

```diff
@@ -642,6 +645,20 @@ func aggregateDatapoints(
 				}
 			}
 			dp.Value = value
+		case AggregationMethodAvg:
+			var mean float64
+			for _, dp := range dps {
+				if dp.Value.IntValue != nil {
+					mean += float64(*dp.Value.IntValue)
+				}
+				if dp.Value.DoubleValue != nil {
+					mean += *dp.Value.DoubleValue
+				}
+			}
+			mean /= float64(len(dps))
+			dp.Value = sfxpb.Datum{
+				DoubleValue: &mean,
```

Comment:
DoubleValue: sum / float64(len(dps))

Reply:
`DoubleValue` requires a `*float64` though. Hence I used `mean` to sum all the values and then compute the average using the same variable. @bogdandrutu - I think to incorporate the two comments here we would need something like this, using an additional variable? Are you concerned about the readability of the current approach? 

```
var sum float64
for _, dp := range dps {
	if dp.Value.IntValue != nil {
	      sum += float64(*dp.Value.IntValue)
        }
	if dp.Value.DoubleValue != nil {
		sum += *dp.Value.DoubleValue
	}
}
mean := sum / float64(len(dps))
dp.Value = sfxpb.Datum{
	DoubleValue: &mean,
}
```